### PR TITLE
kinder: roadmap update

### PR DESCRIPTION
**Pull request by fabriziopandini**
Small updates to kinder roadmap and docs

/assign @neolit123
/area kinder
/kind documentation
/priority backlog
/milestone next

### CODE REVIEW

File: kinder/doc/kind-kinder.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 [kind](https://github.com/kubernetes-sigs/kind) is awesome!
 
-kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
+kinder shares the same core ideas o kind, but, in agreement with the kind team, was developed
```
Comment (neolit123) [inline kinder/doc/kind-kinder.md]: kinder shares the same core ideas o kind, but, in agreement with the kind team, was developed
->
kinder shares the same core ideas of kind, but in agreement with the kind team, was developed

File: kinder/roadmap.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
 
 This document outlines some goals, non-goals, and future aspirations for kinder.
 
-kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
+kinder is heavily inspired by [kind](https://github.com/kubernetes-sigs/kind), but, in agreement with the kind team,
+was developed as a separated tool with the goal to explore new kubeadm-related use cases, share issues and solutions, and
+ultimately contribute back new features.
```
Comment (neolit123) [inline kinder/roadmap.md]: [1]

Comment (fabriziopandini) [reply kinder/roadmap.md]: Uhmm... I don't see the same typo there was in [1] here (o-->of). Am I missing something?

Comment (neolit123) [reply kinder/roadmap.md]: i though the same typo was present.

**Pull request merged**